crates/bevy_ecs/src/change_detection.rs

```diff
@@ -41,6 +41,9 @@ pub trait DetectChanges {
     fn set_changed(&mut self);
 }
 
+// TODO: Add some nice docs here
```

Comment:
If someone would contribute to this point, it'd be highly appreciated

Reply:
```rust
/// When implemented for a component or resource type, this trait allows end users to manually bypass change detection within the ECS using the [`get_untracked`] method on `Query` and `World`.
```

I *think* that's where those methods live?

Reply:
I don't see the `get_untracked` methods in [`Query`](https://github.com/bevyengine/bevy/blob/cf48132efd107bbca85aaea00589a01457ef4872/crates/bevy_ecs/src/system/query.rs) or [`World`](https://github.com/bevyengine/bevy/blob/cf48132efd107bbca85aaea00589a01457ef4872/crates/bevy_ecs/src/world/mod.rs).

Are they found elsewhere? Or are they part of an unmerged PR?

Reply:
It was a "please add these methods too" comment for @TheRawMeatball ;)